network-area-diagram/src/main/java/com/powsybl/nad/svg/iidm/DefaultLabelProvider.java

```diff
@@ -94,7 +93,14 @@ public String getBusDescription(BusNode busNode) {
 
     @Override
     public List<String> getVoltageLevelDetails(VoltageLevelNode vlNode) {
-        return Collections.emptyList();
+        List<String> voltageLevelDetails = new ArrayList<>();
+        long activeProduction = Math.round(network.getVoltageLevel(vlNode.getEquipmentId()).getGeneratorStream().mapToDouble(generator -> !Double.isNaN(-generator.getTerminal().getP()) ? -generator.getTerminal().getP() : 0).sum());
+        long reactiveProduction = Math.round(network.getVoltageLevel(vlNode.getEquipmentId()).getGeneratorStream().mapToDouble(generator -> !Double.isNaN(-generator.getTerminal().getQ()) ? -generator.getTerminal().getQ() : 0).sum());
+        long activeConsumption = Math.round(network.getVoltageLevel(vlNode.getEquipmentId()).getLoadStream().mapToDouble(load -> !Double.isNaN(load.getTerminal().getP()) ? load.getTerminal().getP() : 0).sum());
+        long reactiveConsumption = Math.round(network.getVoltageLevel(vlNode.getEquipmentId()).getLoadStream().mapToDouble(load -> !Double.isNaN(load.getTerminal().getQ()) ? load.getTerminal().getQ() : 0).sum());
+        voltageLevelDetails.add(String.format("Production: %d MW / %d MVAR", activeProduction, reactiveProduction));
```

Comment:
maybe do something special for zero values? "no production/consumption"

Reply:
When there is no data, nothing is displayed.